From 236b56b10cb0e4adb32653788b4ea11a2b5ebb3d Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Mon, 24 Feb 2025 00:41:20 -0600
Subject: [PATCH] build: Add python_requires lower bound

* Python 3.7 is already effectively enforced by the pyhf requirement, but this
  metadata is crucial for installers to not get incorrect versions that are
  broken at install.
---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.py b/setup.py
index f7ac807ed..ee8d194cb 100755
--- a/setup.py
+++ b/setup.py
@@ -128,6 +128,7 @@ setup(
                            'runSModelS.py=smodels.tools.runSModelS:main',
                            'smodelsTools.py=smodels.tools.smodelsTools:main' ]
     },
+    python_requires='>=3.7',
     install_requires=requirements(),
     data_files=dataFiles() ,
     description=("A tool for interpreting simplified-model results from the "
-- 
2.48.1

